lnwallet/channel.go

```diff
@@ -728,6 +728,9 @@ func (lc *LightningChannel) evaluateHTLCView(view *htlcView, ourBalance,
 		if entry.EntryType == Add {
 			continue
 		}
+		if entry.EntryType == Settle && !remoteChain {
```

Comment:
The `entry.EntryType == Settle` fragment isn't needed as this section will only be executed iff `entry.EntryType == Settle` due to the within the loop. 

Also, the log of mutating the channel state's `TotalSatoshis[Received | Sent]` is _mostly_ correct. 

This is due to the value being mutated each time a new view is evaluated. Therefore, value can be skewed in the future if/when we allow multiple in-flight commitments. This can arise since if a party in this scenario initiates 3 updates at once (before we respond), then update (as written now) will mutate the `channelState` incorrectly, causing the wrong value to be written to disk as we revoke our states. Note that this doesn't affect the correctness of the commitment protocol as this value is just used for metrics atm.  

So I think it's fine for the moment, but we may want to re-visit it in the future.